go.mod

```diff
@@ -11,6 +11,9 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk v1.10.0
 	github.com/microsoft/azure-devops-go-api/azuredevops v1.0.0-b3
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
```

Comment:
Why the changes to the go.mod?

Reply:
should be fine now.

Reply:
@EliiseS Good to merge?

Reply:
Yup! I was just running the integration tests :)

Reply:
@C123R Are you able to press the merge button? If not I'll do it :)

Reply:
Nope, I am missing write access!